tests/main.test.ts

```diff
@@ -34,3 +42,16 @@ test("will fail if no zip found", async () => {
 
   expect(tExec).toThrowError()
 })
+
+test("get verbose flag successfully", async () => {
+  env["INPUT_KEYS"] = `{"chrome": {}}`
+  env["INPUT_ZIP"] = "artifacts.zip"
+  env["INPUT_VERBOSE"] = "true"
+
+  const options: ExecFileSyncOptions = {
+    env
+  }
+
+  const output = execFileSync(execPath, [ip], options).toString("utf-8")
```

Comment:
Curious - why is this called `ip`? Initial point?

Reply:
Renamed it to indexScript